src/megaapi_impl.cpp

```diff
@@ -4511,9 +4511,19 @@ void MegaApiImpl::unlink_result(handle h, error e)
 void MegaApiImpl::fetchnodes_result(error e)
 {
 	MegaError megaError(e);
```

Comment:
Indentation doesn't seem to be correct here. The line `MegaError megaError(e);` (as well as some others I'd suspect) is indented with a TAB.